tests/unit_tests/charts/test_client_processing.py

```diff
@@ -2653,3 +2653,172 @@ def test_pivot_multi_level_index():
 | ('Total (Sum)', '', '')           |            210 |            105 |              0 |
     """.strip()
     )
+
+
+def test_apply_client_processing_csv_format_preserves_na_strings():
+    """
+    Test that apply_client_processing preserves "NA" strings
+    when REPORTS_CSV_NA_NAMES is set to empty list.
+    This ensures that scheduled reports can be configured to
+    preserve strings like "NA" as literal values.
```

Comment:
Nit, [multiline docstrings should have a 1 line summary, empty line, additional lines](https://peps.python.org/pep-0257/#multi-line-docstrings).

```suggestion
    Test that apply_client_processing preserves "NA" when REPORTS_CSV_NA_NAMES is [].

    This ensures that scheduled reports can be configured to
    preserve strings like "NA" as literal values.
```